h/views/client.py

```diff
@@ -61,8 +94,12 @@ def _app_html_context(assets_env, api_url, service_url, sentry_public_dsn,
             'googleAnalytics': ga_client_tracking_id
         })
 
-    app_css_urls = assets_env.urls('app_css')
-    app_js_urls = assets_env.urls('app_js')
+    if client_url:
+        app_js_urls = [client_url]
```

Comment:
I thought you were going to use `request.route_path('embed')` here to avoid having to do an HTTP call every time we render `app.html` -- which is notably not served with sensible cache headers at the moment.